server/cache/two_queue.go

```diff
@@ -0,0 +1,163 @@
+// Copyright 2017 PingCAP, Inc.
```

Comment:
em, I think we must include golang-lru license in this file.

Reply:
How to do that? just include golang-lru/LICENSE content?

Reply:
yes, include golang-lru license